sdk/core/Azure.Core/src/Shared/BearerTokenChallengeAuthenticationPolicy.cs

```diff
@@ -153,138 +142,25 @@ private async Task AuthenticateRequestAsync(HttpMessage message, TokenRequestCon
                 headerValue = _accessTokenCache.GetHeaderValueAsync(message, context, async).EnsureCompleted();
             }
 
-            message.Request.SetHeader(HttpHeader.Names.Authorization, headerValue);
+            message.Request.Headers.SetValue(HttpHeader.Names.Authorization, headerValue);
         }
 
         private static string? GetClaimsChallenge(Response response)
```

Comment:
Is this method still needed here, or can it be removed since it's being handled in `ARMChallengeAuthenticationPolicy`?

Reply:
Nope - missed that, thanks.